add_nyt_bestseller_tag.py

```diff
@@ -119,41 +117,39 @@ def __process_bestseller_group_record(self, bestseller_group_record,
                 if bstslr_edition:
                     self.__process_found_bestseller_edition(bstslr_record_isbn,
                                                             bstslr_edition,
-                                                            new_tags, comment,
+                                                            new_tags,
                                                             job_results)
                 else:
                     self.logger.info(
                         'The edition {} doesnt exist in OL, importing'
                             .format(bstslr_record_isbn))
                     self.__request_book_import_by_isbn(bstslr_record_isbn)
-                    job_results['books_imported'] = \
-                        job_results['books_imported'] + 1
+                    job_results['books_imported'] += 1
             except SystemExit:
                 self.logger.info('Interrupted the bot '
                                  'while processing ISBN {}'
                                  .format(bstslr_record_isbn))
-                job_results['isbns_failed'] = \
-                    job_results['isbns_failed'] + 1
+                job_results['isbns_failed'] += 1
                 self.__save_job_results(job_results)
                 raise
             except:
                 self.logger.exception('Failed to process ISBN {}'
                                       .format(bstslr_record_isbn))
-                job_results['isbns_failed'] = job_results['isbns_failed'] + 1
+                job_results['isbns_failed'] += 1
 
     def run(self) -> None:  # overwrites the AbstractBotJob run method
         self.dry_run = self.args.dry_run
+        self.limit = None
```

Comment:
Hm is this what we have to do to get around limit when doing self.save instead of work.save?

Reply:
Yes, the self.save function checks limit if it is not a dry_run, so I had to reset that check (alternatively for reviews we could probably set it to the len of the reviews array)